aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/RemoveDefaultsDecorator.kt

```diff
@@ -25,6 +25,32 @@ class RemoveDefaultsDecorator : ClientCodegenDecorator {
     // Service shape id -> Shape id of each root shape to remove the default from.
     // TODO(https://github.com/smithy-lang/smithy-rs/issues/3220): Remove this customization after model updates.
     private val removeDefaults = mapOf(
+        "com.amazonaws.amplifyuibuilder#AmplifyUIBuilder".shapeId() to setOf(
```

Comment:
for a little less visual noise I'd probably delete `.shapeId()` in all of these and do `.map { k, v -> k.shapeId() to v.shapeId() }` at the end.